packages/react-components/src/components/AppFrame/AppFrame.stories.tsx

```diff
@@ -0,0 +1,170 @@
+
+export default {
+  title: 'Components/AppFrame',
+  component: AppFrame,
+  parameters: {
+    layout: 'fullscreen',
+  },
+  subcomponents: {
+    Navigation,
+    NavigationItem,
+    NavigationGroup,
+    SideNavigation,
+    SideNavigationItem,
+    SideNavigationGroup,
+  },
+};
+
+const navigationItems = [
+  'home',
+  'chats',
+  'engage',
+  'archives',
+  'tickets',
+  'team',
+  'reports',
+  'apps',
+  'billing',
+  'settings',
+  'news',
+];
+const navigationItemsIcons = [
+  Icons.LiveChatMono,
+  Icons.Messages,
+  Icons.Automation,
+  Icons.Archives,
+  Icons.Tickets,
+  Icons.People,
+  Icons.Report,
+  Icons.Apps,
+];
+const secondaryNavigationIcons = [
+  Icons.CreditCardOutline,
+  Icons.Settings,
+  Icons.Notifications,
+];
+const SectionsWithToggle = ['chats', 'engage', 'archives'];
+
+export const Default = (): React.ReactElement => {
+  const [activeItem, setActiveItem] = React.useState('home');
+  const [activeSubItem, setActiveSubItem] = React.useState(0);
+  const [topBarVisible, setTopBarVisible] = React.useState(true);
+
+  const getSubNav = () => {
+    switch (activeItem) {
+      case 'chats':
+        return getChatsMenu(activeSubItem, setActiveSubItem);
+      case 'engage':
+        return getEngageSubMenu(activeSubItem, setActiveSubItem);
+      case 'archives':
+        return getArchivesSubMenu(activeSubItem, setActiveSubItem);
+      default:
+        return null;
+    }
+  };
+
+  return (
+    <AppFrame
+      navigation={
+        <Navigation>
+          <NavigationGroup scrollable>
+            {navigationItems.slice(0, 8).map((item, index) => (
+              <NavigationItem
+                key={item}
+                id={item}
+                label={item.charAt(0).toUpperCase() + item.slice(1)}
+                icon={<Icon source={navigationItemsIcons[index]} />}
+                onClick={(e, id) => {
+                  e.preventDefault();
+                  setActiveItem(id);
+                }}
+                isActive={activeItem === item}
+                badge={getBadgeContent(item)}
+                url="#"
+              />
+            ))}
+          </NavigationGroup>
+          <NavigationGroup>
+            {navigationItems.slice(8, 11).map((item, index) => (
+              <NavigationItem
+                key={item}
+                id={item}
+                label={item.charAt(0).toUpperCase() + item.slice(1)}
+                icon={<Icon source={secondaryNavigationIcons[index]} />}
+                onClick={(e, id) => {
+                  e.preventDefault();
+                  setActiveItem(id);
+                }}
+                isActive={activeItem === item}
+                url="#"
+              />
+            ))}
+            <NavigationItem
+              id="user"
+              label="User"
+              disableOpacity
+              disableTooltip
+              icon={
+                <Tooltip
```

Comment:
Non-blocking: it would be so cool to have tooltips out of the box, with no need for defining them each time

Reply:
`NavigationItem` has built-in tooltips displaying the label.
This is only an example of how to create the navigation element with its own tooltip (like Avatar with profile menu in the AA)

Reply:
Gotcha, that's great. Resolving